telegram_bot.py

```
@@ -565,6 +602,76 @@ async def _process_callback(bot_token: str, callback: dict) -> None:
+        try:
+            orchestrator.approve(run_id, approved_by=f"telegram:{user_id}")
+        except KeyError:
+            await _answer_callback(bot_token, callback_id, "Run not found.")
+            await _edit_message(
+                bot_token, chat_id, message_id,
+                f"⚠️ Run `{run_id}` not found — it may have expired or already been resolved.",
+            )
+            return
+        except ValueError as exc:
+            await _answer_callback(bot_token, callback_id, "Already resolved.")
+            await _edit_message(bot_token, chat_id, message_id, f"ℹ️ Run `{run_id}`: {exc}")
```

Ruff (0.15.15)

[warning] String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)